index.bs

```diff
@@ -313,7 +313,7 @@ urlPrefix: https://w3c.github.io/FileAPI/#dfn-; type: dfn;
 
 	<h3 id="integration-with-other-scripts-and-events">Integration with other scripts and events</h3>
 
-		<h4 id="allow-modify-clipboard">Event handlers that are <dfn>allowed to modify the clipboard</dfn></h4>
+		<h4 id="allow-modify-clipboard">Event handlers that are <dfn data-export>allowed to modify the clipboard</dfn></h4>
```

Comment:
In BS, I think it's just "export" no? https://tabatkins.github.io/bikeshed/#dfn-export

```suggestion
		<h4 id="allow-modify-clipboard">Event handlers that are <dfn export>allowed to modify the clipboard</dfn></h4>
```

Reply:
It's `data-export` [here](https://tabatkins.github.io/bikeshed/#dfn-contract). @tabatkins?

Reply:
Sorry for not seeing this earlier. I already merged a fix for this with just "export", which I believe is correct here because this export isn't part of the data model.

Reply:
I saw your commit, and `export` just is translated to `data-export` there. I think @tabatkins can clarify what's going on there and the documentation.

Reply:
Yeah, in a bs source file you can just write `export`. It gets turned into `data-export` in the output, so that it's not an invalid attribute.